### PR TITLE
Test member page data

### PR DESCRIPTION
**Pull request by ondenman**
Tests scraper output against YAML file.
MemberPage#to_h should match the data specified in the YAML.

Part of everypolitician/everypolitician#600 (Ensure every scraper that has been ported to Scraped has a basic YAML test for at least one person)

### CODE REVIEW

File: test/data/664.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+:url: http://www.nass.gov.ng/mp/profile/664
+:to_h:
+  :id: '664'
+  :title: Hon.
+  :shorter_name: BELLO ABDULLAHI
+  :name: BELLO ABDULLAHI
+  :constituency: Okene/Ogori-Magogo
+  :state: Kogi State
+  :chamber: House of Representatives
+  :position: ''
```
Comment (tmtmtmtm) [inline test/data/664.yml]: Again, it's worth making sure that every field is covered by at least one test case.

File: Rakefile [hunk 2]
```diff
@@ -3,4 +3,8 @@ require 'rubocop/rake_task'
 
 RuboCop::RakeTask.new
 
-task default: %w(rubocop)
+require 'scraper_test'
+ScraperTest::RakeTask.new.install_tasks
+
+task test: 'test:data'
+task default: %w(rubocop test)
```
Comment (tmtmtmtm) [inline Rakefile]: this looks like you're running the old version of rubocop here. Is the plan to update that separately?

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]

File: Rakefile (hunk 2 above)
Comment (ondenman) [reply Rakefile]: I've opened a PR for this. https://github.com/everypolitician-scrapers/nigeria-national-assembly/pull/18

**Comment by ondenman** [comment]
Github closed this PR after I rebased and force pushed. Not sure why.

I've opened a new PR: https://github.com/everypolitician-scrapers/nigeria-national-assembly/pull/20

**Pull request closed without merging**